docs/setup/native.md

```diff
@@ -59,7 +59,8 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
     - `echo "DatabaseMirror database.clamav.net" > freshclam.conf`
     - `freshclam -v`
 1. Install [pdftk](https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/pdftk_server-2.02-mac_osx-10.11-setup.pkg)
-1. Install gem dependencies: `cd vets-api; bundle install`
+1. Install gem dependencies: `cd vets-api; bundle install --without production`
```

Comment:
This turns out to break `bundle outdated` which doesn't have a way to skip certain gems that I can find.

Reply:
Blocked it until I remove this. The local settings is useful, but this turns out to cause other problems with bundling.